docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -446,9 +444,95 @@ Where,
 Iron Fortification
 ~~~~~~~~~~~~~~~~~~
 
+Population Coverage Data and Coverage Algorithm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The baseline coverage for iron fortification is the same as the baseline 
+coverage for folic acid fortification, as described below_.
```

Comment:
We should clarify exactly what this means, as James requested yesterday.

My assumption is that exactly the same people will get iron fortification as will get folic acid fortification (as opposed to, e.g. independently choosing the same proportion of the population to get each).

@alibow @Ninicorn @aflaxman Does that seem like the appropriate strategy?

@collijk Does that clarify what you were asking about?

Reply:
Yes, seems appropriate to me. Feel free to directly update when you get confirmation.

Reply:
That's another good question. I wasn't totally clear what James was asking. I think that's a question for @aflaxman since he suggested modeling them separately as well.

Reply:
Since Iron and Folic acid are often considered together, I propose that:
(1) *baseline fortification be correlated* so that exactly the same people get iron fortification as get folic acid fortification;
(2) we include scenarios for (a) scale-up of Iron Fortification coverage (b) scale-up of Folic Acid Fortification; and (c) simultaneous scale-up of Iron and Folic Acid Fortification (in scenario [c] exactly the same people get iron fortification as get folic acid fortification)

Reply:
So to confirm, my interpretation is: 1) *baseline* coverage will be exactly correlated for iron and folic acid. 2) *intervention* coverage will not be correlated for scenarios a and b, and will be exactly correlated for scenario c.

Reply:
I will update the documentation here to reflect this unless I hear otherwise! Thanks :)

Reply:
confirmed!

Reply:
Great, thanks! This is now documented in the latest commit.